samples/server/petstore/php-mezzio-ph-modern/composer.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": "git_user_id/git_repo_id",
+    "description": "",
+    "license": "unlicense",
+    "version": "1.0.0",
+    "type": "project",
+    "require": {
+        "php": "^8.0",
```

Comment:
Does it install properly with PHP 7.3? I thought we need `^7.3 || ^8.0` version constraint.

Reply:
Nope, samples in `samples/server/petstore/php-mezzio-ph-modern` are generated with `modern=true` (it is a new CLI option for `php-mezzio-ph`) and this code can run only on PHP 8.0 . Samples in `samples/server/petstore/php-mezzio-ph` are generated with `modern=false` and [its `composer.json`](https://github.com/OpenAPITools/openapi-generator/blob/master/samples/server/petstore/php-mezzio-ph/composer.json) contains valid `"php": "^7.3 || ^8.0"`.